v21.2/use-changefeeds.md

```diff
@@ -197,35 +197,33 @@ The following sections provide information on Avro usage with CockroachDB change
 
 Below are clarifications for particular SQL types and values for Avro changefeeds:
 
-- [Decimals](decimal.html) must have precision specified.
-- [`BIT`](bit.html) and [`VARBIT`](bit.html) types are encoded as arrays of 64-bit integers.
-
-  {% include {{ page.version.version }}/cdc/avro-bit-varbit.md %}
+{% include {{ page.version.version }}/cdc/avro-limitations.md %}
 
 ### Avro types
 
 Below is a mapping of CockroachDB types to Avro types:
 
 CockroachDB Type | Avro Type | Avro Logical Type
 -----------------+-----------+---------------------
-[`INT`](int.html) | [`LONG`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
+[`ARRAY`](array.html) | [`ARRAY`](https://avro.apache.org/docs/1.8.1/spec.html#schema_complex) |
+[`BIT`](bit.html) | Array of [`LONG`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
 [`BOOL`](bool.html) | [`BOOLEAN`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
+[`BYTES`](bytes.html) | [`BYTES`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
```

Comment:
Question: would it be useful to include `BYTEA` and `BLOB` here in some fashion, somewhat similar to `TIMESTAMP` and `TIMESTAMPZ`? Maybe, as aliases rather than a variant, same line unlinked is more appropriate? Unsure.

```
[`BYTES`](bytes.html), `BYTEA`, `BLOB` | [`BYTES`](avrolink)
```

Love the alpha reordering, thank you!

Reply:
Cool nice changes! I think it makes sense to just additional lines for the aliases like the time related types. TFTR!